src/pizza-guy.js

```diff
@@ -12,8 +12,10 @@ module.exports = {
     if (!Array.isArray(list)) {
       throw Error('The list must be an array');
     }
-    if (!list.every((item) => typeof item === 'string')) {
-      throw Error('The list must contains just strings');
+    if (!list.every(
+        (item) => typeof item === 'string' || (typeof item === 'object' && item.url)
```

Comment:
Since we're only checking the existence of `item.url` we should do
```js
item.hasOwnProperty('url')
```
by doing that we can get rid of the extra condition there.